samples/app-server/bin/start.sh

```diff
@@ -13,4 +13,4 @@ JAVA_OPTIONS=$JAVA_OPTIONS\ -Xmx512m
 JAVA_OPTIONS=$JAVA_OPTIONS\ -Djava.security.egd=file:/dev/./urandom
  
 # run Java
-java -jar start.jar $JAVA_OPTIONS
+java --add-opens=java.base/java.lang=ALL-UNNAMED -jar start.jar $JAVA_OPTIONS
```

Comment:
Add argument to `JAVA_OPTIONS` variable instead?

Reply:
Done. I also fixed the position of the JAVA_OPTIONS variable on the command line so that it can take effect at all.